bitnami/kafka/templates/kafka-provisioning.yaml

```diff
@@ -140,32 +152,21 @@ spec:
         {{- include "common.tplvalues.render" (dict "value" .Values.provisioning.sidecars "context" $) | nindent 8 }}
         {{- end }}
       volumes:
-        {{- if or .Values.config .Values.existingConfigmap }}
-        - name: kafka-config
+        - name: kafka-client-config
           configMap:
-            name: {{ include "kafka.configmapName" . }}
-        {{- end }}
+            name: {{ include "kafka.client.configmapName" . }}
```

Comment:
Delete `kafka-client-config` (Github UI won't let me suggest here for some reason)